src/main/java/frc/robot/Intake.java

```diff
@@ -0,0 +1,63 @@
+package frc.robot;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+/**
+ * Add&named motors in Intake.
+ */
+public class Intake extends SubsystemBase {
+    private TalonSRX motor = new TalonSRX(Ports.Intake.MOTOR);
+    private Solenoid piston = new Solenoid(Ports.Intake.SOLENOID);
```

Comment:
Better to call this "retractor" or something similar, so that it's clear what is the piston's purpose.